sunpy/net/helioviewer.py

```diff
@@ -193,9 +261,15 @@ def download_png(self, date, image_scale, layers, directory=None,
             (Optional) Width of the image in pixels (Maximum: 1920).
         height : int
             (Optional) Height of the image in pixels (Maximum: 1200).
-        watermark
-            (Optional) Whether or not the include the timestamps and the
-            Helioviewer.org logo in the image (Default=True).
+        watermark : bool
+            (Optional) Optionally overlay a watermark consisting of a
+            Helioviewer logo and the datasource abbreviation(s) and
+            timestamp(s) displayed in the screenshot.
+        display : bool
+            (Optional) Set to `true` to directly output binary PNG image data.
+            Default is `false` (which outputs a JSON object).
```

Comment:
Don't we want this to be set internally? If it outputs a JSON object do we still get a png to download?